src/main/java/net/rpgtoolkit/common/assets/AnimationFrame.java

```diff
@@ -56,16 +56,6 @@ public AnimationFrame(String frameName, long transparentColour, String frameSoun
       } catch (IOException e) {
         e.printStackTrace();
       }
-    } else {
-      // Loads the tileset and fetches the correct tile
-      frameName = frameName.toLowerCase();

```

Comment:
They are used in Animations, they are made up of AnimationFrames, easier just to say they are created from png, jpgs, gifs. 

Having the option to use individual tiles from tilesets over complicates things.